lib/gl_exception_notifier.rb

```diff
@@ -65,6 +48,32 @@ def last_breadcrumb
 
     private
 
+    def exceptionable?(obj)
+      obj.is_a?(Exception) ||
+        (obj.respond_to?(:ancestors) && obj.ancestors.include?(Exception))
+    end
+
+    def capture_exception(args)
+      error_client.capture_exception(*args)
+    end
+
+    def capture_message(args)
+      if args.first.is_a?(String)
+        message = args.first
+        extra = args.length == 2 && args.last.is_a?(Hash) ? args.last : { parameters: args.drop(1) }
+      else
+        message_info = if args.first.is_a?(Hash)
```

Comment:
Here is the functional change: make it clear where the issue is coming from (rather than the inscrutable `'Unknown parameter set'`